src/session/session_compact.c

```diff
@@ -213,7 +233,7 @@ __compact_worker(WT_SESSION_IMPL *session)
     /*
      * Perform an initial checkpoint (see this file's leading comment for details).
      */
-    WT_ERR(__wt_session_blocking_checkpoint(session, true, session->compact->max_time));
+    WT_ERR(__compact_checkpoint(session));
```

Comment:
I think this means that `__w_session_blocking_checkpoint` can be static to its location, there's only a single caller now.

@kommiharibabu modeled his work there on top of how compact worked, so there also may be some changes to make there, that is, if compact was wrong (or at least, could be improved), maybe that code should be updated in the same way.

Reply:
Good point. The only (probably not a viable) reason I left `__wt_session_blocking_checkpoint ` in public API is that there could be other, non-WT, consumers who use it. But I'm happy to make it static if you think it's OK.

Reply:
Yes, you can change it into a local function as it is used only in one place and also that function can be simplified as most of the options are not required as per the new code.

Reply:
I've renamed `__wt_session_blocking_checkpoint` to `__session_blocking_checkpoint` and made it static. `force` and `seconds` parameters are not necessary anymore, so I removed them and the code associated with them. Thank you!